doc/sources/memory-requirements.rst

```diff
@@ -0,0 +1,42 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _memory_requirements:
+
+###################
+Memory requirements 
+###################
+
+By default, algorithms in |intelex| run in multi-threaded mode, which uses all available threads. 
+This may lead to optimized algorithms consuming more RAM than the corresponding stock scikit-learn algorithms.
+
+.. list-table::
+   :widths: 10 30 30
+   :header-rows: 1
+   :align: left
+
+   * - Algorithm
+     - Single-threaded
+     - Multi-threaded
+   * - SVM
+     - Both Scikit-learn and |intelex| consume approximately the same amount of RAM.
```

Comment:
Do we have some where information in doc about how to run the algorithms in single thread-mode? If yes, it might be useful to add link here.

Reply:
We have function to change number of threads - [daalinit](https://github.com/intel/scikit-learn-intelex/blob/master/generator/wrapper_gen.py#L165).
But no information about this in doc